src/emm/emm_cli.py

```diff
@@ -211,5 +220,15 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_context_width=100))
+@click.argument("commit", nargs=-1)
+@click.option("--directory", default=None, help="Directory to execute command at.")
```

Comment:
Why does this take a directory as an option? All the other commands are expected to be run at the base of the repository, why is this one different?

Reply:
I misunderstood the git-related functions' `directory: Optional[Path]`. The command we issue should not allow user pass directory just add and commit changes on the base repo. We only apply directory argument to enabled modules only.